FunctionalTableDataTests/TableSectionChangeSetTests.swift

```diff
@@ -63,6 +63,31 @@ class TableSectionChangeSetTests: XCTestCase {
 		XCTAssertEqual(changes.deletedSections, IndexSet())
 	}
 
+	func testSectionNotReloadedWithEqualHeaders() {
+		let oldItems: [TableSection] = [TableSection(key: "section1", header: TestHeader(state: TestHeaderState(data: "state1")))]
+		let newItems: [TableSection] = [TableSection(key: "section1", header: TestHeader(state: TestHeaderState(data: "state1")))]
+		let changes = TableSectionChangeSet(old: oldItems, new: newItems, visibleIndexPaths: [])
+
+		XCTAssertTrue(changes.isEmpty)
+		XCTAssertEqual(changes.movedSections, [])
+		XCTAssertEqual(changes.insertedSections, IndexSet())
+		XCTAssertEqual(changes.reloadedSections, IndexSet())
+		XCTAssertEqual(changes.deletedSections, IndexSet())
+	}
+
+	func testSectionReloadedWithInequalHeaders() {
+		let oldItems: [TableSection] = [TableSection(key: "section1", header: TestHeader(state: TestHeaderState(data: "state1")))]
+		let newItems: [TableSection] = [TableSection(key: "section1", header: TestHeader(state: TestHeaderState(data: "state2")))]
+		let changes = TableSectionChangeSet(old: oldItems, new: newItems, visibleIndexPaths: [])
+
+		XCTAssertFalse(changes.isEmpty)
+		XCTAssertEqual(changes.count, 1)
+		XCTAssertEqual(changes.movedSections, [])
+		XCTAssertEqual(changes.insertedSections, IndexSet())
+		XCTAssertEqual(changes.reloadedSections, IndexSet([0]))
+		XCTAssertEqual(changes.deletedSections, IndexSet())
+	}
+
```

Comment:
we are missing footers tests 🙂

Reply:
Tests updated to add rows, and footer tests added